src/main/java/commands/AddTodoCommand.java

```diff
@@ -0,0 +1,41 @@
+package commands;
+
+import storage.Storage;
+import tasklist.TaskList;
+import tasks.Task;
+import ui.Ui;
+
+/**
+ * Represents a command to add a todo task to the task list.
+ */
+public class AddTodoCommand implements Command {
+
+    /**
+     * The regular expression pattern for matching the format of a todo command.
+     */
+    public static final String TODO_PATTERN = "^(todo)\\s+.+";
```

Comment:
I like the use of a constant field here, and the naming follows coding standard too!